<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>421</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>421</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forwardButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>⬆</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>30</number>
   </property>
   <property name="autoRepeatInterval">
    <number>30</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="flipLeftRightCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
 	width: 20px;
   height: 20px;               
}</string>
   </property>
   <property name="text">
    <string>Flip left/right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>⬆</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>30</number>
   </property>
   <property name="autoRepeatInterval">
    <number>30</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="flipFrontBackCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
	width: 20px;
	height: 20px;               
}</string>
   </property>
   <property name="text">
    <string>Flip front/back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>⬅</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>30</number>
   </property>
   <property name="autoRepeatInterval">
    <number>30</number>
   </property>
  </widget>
  <widget class="QPushButton" name="backwardButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>⬇</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>30</number>
   </property>
   <property name="autoRepeatInterval">
    <number>30</number>
   </property>
  </widget>
  <widget class="QPushButton" name="rightButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>➡</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>30</number>
   </property>
   <property name="autoRepeatInterval">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>⬇</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>30</number>
   </property>
   <property name="autoRepeatInterval">
    <number>30</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
